src/BenchmarksApps/Kestrel/PlatformBenchmarks/BenchmarkApplication.Caching.cs

```diff
@@ -10,7 +11,31 @@ public partial class BenchmarkApplication
     {
         private async Task Caching(PipeWriter pipeWriter, int count)
         {
-            OutputMultipleQueries(pipeWriter, await Db.LoadCachedQueries(count));
+            OutputCachedWords(pipeWriter, await Db.LoadCachedQueries(count));
+        }
+
+        private static void OutputCachedWords(PipeWriter pipeWriter, CachedWorld[] rows)
+        {
+            var writer = GetWriter(pipeWriter, sizeHint: 160 * rows.Length); // in reality it's 152 for one
+
+            writer.Write(_dbPreamble);
+
+            var lengthWriter = writer;
+            writer.Write(_contentLengthGap);
+
+            // Date header
+            writer.Write(DateHeader.HeaderBytes);
+
+            writer.Commit();
+
+            Utf8JsonWriter utf8JsonWriter = t_writer ??= new Utf8JsonWriter(pipeWriter, new JsonWriterOptions { SkipValidation = true });
```

Comment:
One spends too much time in the runtime repository

Reply:
Should the `JsonWriterOption` be cached?

Reply:
I am stupid, instead of copying the `Word` version from multiple queries I could have made it generic...

@sebastienros PTAL one more time